### ISSUE TITLE
Wordpress Login Not Working on WebGL build working in Editor

### ISSUE THREAD

**Issue by XRGeek** [mybadstudios/BadDreamsDemo#1]

**Comment by mybadstudios** [comment]
Wow! This took a long time for me to see! 😮Apologies.

There are two issues with WebGL builds to be aware of:

1. I recently discovered my version of Unity defaults to archiving my builds using the Brotil (or something like that) format and my server (and most likely other people's servers also) simply don't have support for that archive format. The solution is simply to go  into your project settings and change the compression to None or ideally Gzip. Build, upload and rejoice as your game starts to play! :)

2. Unfortunately, the second issue is not fixable. WebGL builds do have the issue of not supporting cross domain database access. If you inspect the source of the webpage the player is failing to load on you will see that your browser is selectively picking and choosing which lines of code it will and will not run. 

Imagine: my code is perfect, you use my code perfectly, Unity builds the WebGL game perfectly, you run the game and Chrome says "Nope, not gonna run this line of code or that line of code, that line has got to go, nope, nope, not that one, not this nor that. Okay the rest I will run. Hope I didn't break anything... and if I did, then it's your fault. Go fix it till I am happy".

That is literally what is happening thanks to CORS. Once you get past CORS you run into the issue that you received a cookie from another website but Unity does not allow you to SEND any cookies from any other domains meaning you can't tell the other website where you just logged in to who you are... 

Simply put, you cannot run this kit across domains. You need to upload WebGL builds to their intended destination domain and run them from that domain. Running them from another domain will NOT work. Your browser will simply break your game after you made sure to build it perfectly.
